File: experiments/020-PythonVsJava/cells/python-pytest_1-Empty_free/2019-2c-parcial2-masked/20260921-213927-34997/workdir/src/surveycrawler/traversable_ground.py
from surveycrawler.ground import Ground


class TraversableGround(Ground):
    """Ground the crawler can stand on, and therefore turn on."""

    # facing

    def turn_crawler_clockwise(self, a_crawler):
        a_crawler.turn_facing_clockwise()

    def turn_crawler_counter_clockwise(self, a_crawler):
        a_crawler.turn_facing_counter_clockwise()

File: experiments/020-PythonVsJava/cells/java-gradle_1-Empty_free/2019-2c-parcial2-masked/20260921-212555-19812/workdir/src/main/java/surveycrawler/TraversableGround.java
package surveycrawler;

public abstract class TraversableGround extends Ground {

    // facing - the crawler can always turn on a ground it can be on

    @Override
    public void turnCrawlerClockwise(SurveyCrawler aCrawler) {
        aCrawler.turnFacingClockwise();
    }

    @Override
    public void turnCrawlerCounterClockwise(SurveyCrawler aCrawler) {
        aCrawler.turnFacingCounterClockwise();
    }
}
